zkevm-circuits/src/copy_circuit.rs

```diff
@@ -250,6 +255,15 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
                 )(meta),
             ]);
             vec![
+                // If a row is not enabled (fixed), it is not in an event.
+                // TODO: not currently possible due to API limitations.
```

Comment:
what does `API limitations` mean ?

Reply:
The Halo2 library has the concept of blinding rows, and it checks that no constraints are enabled on blinding rows. However, that is incorrect in our case and that is a soundness bugs. To fix separately.

Reply:
for blinding rows , q_enable will be false such that constraint not checking on those rows.  why think it is a problem ?

Reply:
Because then nothing is checked and the blinding rows could contain an incorrect event.

Reply:
blinding rows filled with random data, essentially no need to constrain it. it is not specific with copy circuit. all other circuit can have blinding rows with any data.